AutoCollection/Statsbeat.ts

```diff
@@ -148,15 +164,52 @@ class Statsbeat {
         this._retryCount++;
     }
 
+    public async trackNetworkStatsbeatMetrics() {
+        this._getResourceProvider(async () => {
+            let networkProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion,
+                "attach": this._attach,
+                "instrumentation": this._instrumentations,
+            }
+            this._trackRequestDuration(networkProperties);
+            this._trackRequestsCount(networkProperties);
+            await this._sendStatsbeats();
+        });
+    }
+
     public async trackStatsbeatMetrics() {
         this._getResourceProvider(async () => {
-            this._trackRequestDuration();
-            this._trackRequestsCount();
+            let attachProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "rpid": this._resourceIdentifier,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion
+            }
+            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.ATTACH, value: 1, properties: attachProperties });
```

Comment:
attach statsbeat is every 15 mins.  feature is every 24 hours. you might want to have a separate method for attach and one for feature.

Reply:
@heyams For applications that exit before 24 hours, do we send a statsbeat telemetry upon exit?

Reply:
@heyams  Why attach is being sent every 15 minutes?, are you expecting rpId to be updated that often?

Reply:
this is the same interval as the azure metadata service.

Reply:
vm id can get changed frequently, which impacts rpId for vm. other attach scenarios is not an issue.

Reply:
> @heyams For applications that exit before 24 hours, do we send a statsbeat telemetry upon exit?

it will be nice.  currently java doesn't do that.

Reply:
It's not only about "nice". If applications exit, there may be cases where we never get the telemetry. We MUST flush before exit.

Reply:
@hectorhdzg probably not blocking for this pr though

Reply:
@lzchen we flush all telemetry on exit, but we do not create telemetry on exit, we don't want to do that, that will make sense with high priority telemetry, maybe something that actually let users know that the app stopped running, but that doesn't apply with Statsbeats

Reply:
Interesting. Something to bring up at tomorrow's meeting.